### PR TITLE
Add 5.4.4 version history

### PR DESCRIPTION
**Pull request by hflynn**
# What this PR does

Adds the version history for 5.4.4

# Testing this PR

Proofread. To test the rendered content you'll need to manually paste the content into the version history docs page and build it locally - the content will be added to the docs via the autogen build *after* merging.

# Related reading

See https://trello.com/c/0B10cnAO/29-version-history-pr

### CODE REVIEW

File: history.txt [hunk 1]
```diff
@@ -5,6 +5,37 @@
...
 =====================
 
+5.4.4 (March 2018)
+------------------
+
+This is a bug-fix release which also introduces some new functionality.
+
+Improvements include:
+
+-  images can now be filtered by Tag in the center panel
```
Comment (jburel) [inline history.txt]: specify that it is in web

File: history.txt [hunk 2]
```diff
@@ -5,6 +5,37 @@
...
+-  enabled search by "File" and "Tag" annotations separately, as opposed to
+   only being able to search by All annotations
+-  fixed an error when projecting an image with all the channels turned off
+-  fixed parsing of polygons and polyline ROIs so they can be opened in ImageJ
+-  fixed creation of OMERO pyramids for little endian files
+
+Sysadmin improvements include:
+
+-  improved installation documentation for OMERO.web, and OMERO.server on
+   Debian 9 and Centos7
```
Comment (jburel) [inline history.txt]: and Ubuntu 16.04

**Comment by jburel** [comment]
Also to add: (not included in workflows branch)
Users:

* improve message when login failed in insight (#5658)
* improve handling of idle connection in insight see #5650

Devs
* add example to create session for other (#5652 see also #5651)
* add support for more rendering parameters to api (#5592)
* add option to respect specific tile size (#5542)
* add method to load plane using JavaGateway (#5651)

Sysadmin/Users:
* allow to enforce secure connection when importing data (#5645)

File: history.txt [hunk 3]
```diff
@@ -12,7 +12,7 @@ This is a bug-fix release which also introduces some new functionality.
 
 Improvements include:
 
--  images can now be filtered by Tag in the center panel
+-  images can now be filtered by Tag in the center panel of OMERO.web
 -  enabled search by "File" and "Tag" annotations separately, as opposed to
```
Comment (jburel) [inline history.txt]: search by ... is also web

File: history.txt [hunk 4]
```diff
@@ -5,6 +5,46 @@
...
+This is a bug-fix release which also introduces some new functionality.
+
+Improvements include:
+
+-  images can now be filtered by Tag in the center panel of OMERO.web
+-  enabled search by "File" and "Tag" annotations separately in OMERO.web, as 
+   opposed to only being able to search by All annotations
+-  fixed an error when projecting an image with all the channels turned off
+-  fixed parsing of polygons and polyline ROIs so they can be opened in ImageJ
+-  fixed creation of OMERO pyramids for little endian files
```
Comment (mtbc) [inline history.txt]: hyphenate little-endian here

File: history.txt [hunk 5]
```diff
@@ -5,6 +5,46 @@
...
+-  improved error message when login fails for OMERO.insight
+-  improved handling of idle connections in OMERO.insight
+
+Sysadmin improvements include:
+
+-  improved installation documentation for OMERO.web, and OMERO.server on
+   Debian 9, Ubuntu 16.04 and Centos7
```
Comment (mtbc) [inline history.txt]: CentOS 7

File: history.txt [hunk 6]
```diff
@@ -5,6 +5,46 @@
...
+-  added an admin command and script to allow deletion of corrupted pyramids
+   created by a bug introduced with OMERO 5.2 (new uncorrupted pyramids can
+   then be generated)
+-  allowed enforcement of a secure connection when importing data
+
+Developer updates include:
+
+-  improved loading speed of LUT
```
Comment (mtbc) [inline history.txt]: Isn't this a general improvement, not just for developers?

Comment (hflynn) [reply history.txt]: I wondered similarly, I'll move it to the top list

File: history.txt [hunk 7]
```diff
@@ -5,6 +5,46 @@
...
+------------------
+
+This is a bug-fix release which also introduces some new functionality.
+
+Improvements include:
+
+-  images can now be filtered by Tag in the center panel of OMERO.web
+-  enabled search by "File" and "Tag" annotations separately in OMERO.web, as 
+   opposed to only being able to search by All annotations
+-  fixed an error when projecting an image with all the channels turned off
```
Comment (jburel) [inline history.txt]: preview is fixed not the full projection
that's a different bug

**Comment by jburel** [comment]
@hflynn I am working on a fix so the full projection will hopefully be sorted too

**Comment by jburel** [comment]
To add
Improvements:
 * Able to start insight/importer with Java 9
 * Disable projection in insight if all channels are inactive see #5670
 * Fix switch between grid display and thumbnails  display in web #5669

**Comment by jburel** [comment]
To add:
* Adds  commands to the CLI  session plugin, to enable the creation and removal of user sessions. #5614

File: history.txt [hunk 8]
```diff
@@ -5,6 +5,51 @@
...
+
+Sysadmin improvements include:
+
+-  improved installation documentation for OMERO.web, and OMERO.server on
+   Debian 9, Ubuntu 16.04 and CentOS 7
+-  added an admin command and script to allow deletion of corrupted pyramids
+   created by a bug introduced with OMERO 5.2 (new uncorrupted pyramids can
+   then be generated)
+-  allowed enforcement of a secure connection when importing data
+-  added commands to the CLI admin session plugin to enable the creation and
```
Comment (jburel) [inline history.txt]: remove admin

**Comment by jburel** [comment]
@hflynn one minor edit: remove ``admin``
it is the ``sessions`` plugin

File: history.txt [hunk 9]
```diff
@@ -33,7 +33,7 @@ Sysadmin improvements include:
    created by a bug introduced with OMERO 5.2 (new uncorrupted pyramids can
    then be generated)
 -  allowed enforcement of a secure connection when importing data
--  added commands to the CLI admin session plugin to enable the creation and
+-  added commands to the CLI session plugin to enable the creation and
```
Comment (mtbc) [inline history.txt]: "sessions" plural, as in https://docs.openmicroscopy.org/omero/5.4.3/users/cli/sessions.html#multiple-sessions

**Comment by mtbc** [comment]
Thank you, that addresses all my concerns.

**Comment by jburel** [comment]
It matches what was discussed this morning
The link to the correct announcement will be fixed in another PR

**Comment by jburel** [comment]
merging for rc1 prep

**Pull request merged**